history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -1551,6 +1697,10 @@ who having petitioned the king,
 that he would be pleased to cause them to be made freemen,
 obtained the following order from him:
 
+[.embedded-content-document.legal]
+--
+
+[.letter-heading]
 To our trusty and well-beloved our Attorney-General.
 
 James R.
```

Comment:
Again, it seems like this is from the king written by Sunderland.

Reply:
yup, let's go `[ssco]` with it as well.